File: Ostin/CoreData/Item+CoreDataProperties.m
//
//  Item+CoreDataProperties.m
//  
//
//  Created by Denis Kurochkin on 07/10/2016.
//
//

#import "Item+CoreDataProperties.h"

@implementation Item (CoreDataProperties)

+ (NSFetchRequest<Item *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Item"];
}

@dynamic additionalInfo;
@dynamic additionalSize;
@dynamic barcode;
@dynamic boxType;
@dynamic certificationAuthorittyCode;
@dynamic certificationType;
@dynamic collection;
@dynamic color;
@dynamic drop;
@dynamic groupID;
@dynamic itemCode;
@dynamic itemCode_2;
@dynamic itemID;
@dynamic line1;
@dynamic line2;
@dynamic name;
@dynamic priceHeader;
@dynamic size;
@dynamic sizeHeader;
@dynamic storeNumber;
@dynamic subgroupID;
@dynamic trademarkID;

@end

File: Item+CoreDataProperties.swift
//
//  Item+CoreDataProperties.swift
//  
//
//  Created by Denis Kurochkin on 03/10/2016.
//
//

import Foundation
import CoreData


extension Item {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Item> {
        return NSFetchRequest<Item>(entityName: "Item");
    }

    @NSManaged public var itemID: Int32
    @NSManaged public var itemCode: String?
    @NSManaged public var groupID: Int32
    @NSManaged public var subgroupID: Int32
    @NSManaged public var trademarkID: Int32
    @NSManaged public var color: String?
    @NSManaged public var certificationType: String?
    @NSManaged public var certificationAuthorittyCode: String?
    @NSManaged public var itemCode_2: String?
    @NSManaged public var line1: String?
    @NSManaged public var line2: String?
    @NSManaged public var storeNumber: String?
    @NSManaged public var name: String?
    @NSManaged public var priceHeader: String?
    @NSManaged public var sizeHeader: String?
    @NSManaged public var size: String?
    @NSManaged public var additionalSize: String?
    @NSManaged public var additionalInfo: String?
    @NSManaged public var boxType: String?
    @NSManaged public var barcode: String?

}
